ansible/roles/base/tasks/main.yml

```diff
@@ -17,18 +15,23 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
```

Comment:
I know I had this in there, but to be honest, I’m not sure we need it. I think that the virtualenv install covers this. Also, realistically, no one should be pip installing via the command line anyway.